guides/11_other-tutorials/deploying-gradio-with-disco.md

```diff
@@ -0,0 +1,142 @@
+- A domain name that you can configure.
+- A GitHub account.
+- Basic knowledge of the command line.
+
+### Step 1: Create a Server
+
+First, you'll need a server to host your Gradio app. Choose a provider and create a new server with Ubuntu 24.04 as the operating system.
+
+Once your server is up and running, take note of its IP address. You'll need it for the next step.
+
+### Step 2: Configure DNS Settings
+
+Before going further, you need to set up two domain names. Go to your domain registrar's DNS management panel and add these records:
+
+1.  A domain for your Disco server (e.g., `disco.example.com`).
+2.  A domain for your Gradio application (e.g., `gradio.example.com`).
+
+For the server domain, create an **A record** pointing to your server's IP address:
+
+- **Type**: A
+- **Name**: disco
+- **Value**: `<your_server_ip_address>`
+
+For the application domain, create a **CNAME record** pointing to your server domain:
+
+- **Type**: CNAME
+- **Name**: gradio
+- **Value**: `disco.example.com`
+
+DNS changes can take a few minutes to propagate. You can verify that your server domain is resolving to the correct IP address by running `ping disco.example.com`
+
+### Step 3: Test Your Server Connection
+
+Now that your DNS is set up, let's test the SSH connection to your server from your local machine. This ensures you can access it before we hand things over to Disco.
+
+```bash
+# Replace with your server domain
+ssh root@disco.example.com
+```
+
+If the connection is successful, great! **This is the last time you'll need to SSH into this server manually.** Now, exit the SSH session to return to your local machine. This is a crucial step!
+
+```bash
+exit
+```
+
+### Step 4: Install the Disco CLI on Your Local Machine
+
+**Important:** From this point forward, all commands should be run from your **local machine's terminal**. You will not need to SSH into your server again.
+
+Let's install the Disco command-line interface (CLI) on your local machine. This is the tool you'll use to manage your deployments.
+
+```bash
+curl https://cli-assets.letsdisco.dev/install.sh | sh
+```
+
+After the installation is complete, verify it's working by running:
+
+```bash
+disco --version
+```
+
+### Step 5: Initialize Your Server with Disco
+
+Now, from your local machine, let's set up Disco on your server using the domain you configured.
+
+```bash
+# Replace with your server domain
+disco init root@disco.example.com
+```
+
+This command will:
+
+- Connect to your server using SSH.
+- Install Docker.
+- Set up the Disco server.
+- Configure the initial SSL certificate.
+
+> **Note:** Disco will automatically try to use your default SSH keys. If you use a non-standard key, you can specify the path with the `-i` flag, like so: `disco init -i /path/to/your/ssh/key root@disco.example.com`
```

Comment:
This doesn't render well, and looks like this: 

![Screenshot 2025-07-09 at 5 09 12 PM](https://github.com/user-attachments/assets/3fe87354-55a0-4bbb-852c-39303886b406)

Try using 'Tip: ' or 'Warning: ' instead, which will look like this: 

![Screenshot 2025-07-09 at 5 08 34 PM](https://github.com/user-attachments/assets/df6591af-3650-44a5-9cfb-c9f6eab15e2b)

Reply:
```suggestion
Tip: Disco will automatically try to use your default SSH keys. If you use a non-standard key, you can specify the path with the `-i` flag, like so: `disco init -i /path/to/your/ssh/key root@disco.example.com`
```